src/main/java/duke/command/AssignTaskToPatientCommand.java

```diff
@@ -12,15 +14,17 @@
 
 public class AssignTaskToPatientCommand extends Command {
 
+    private String command;
     private PatientTask newPatientTask;
 
-    public AssignTaskToPatientCommand(PatientTask patientTask) {
+    public AssignTaskToPatientCommand(String cmd) {
         super();
-        this.newPatientTask = patientTask;
+        this.command = cmd;
     }
 
     @Override
     public void execute(PatientTaskList patientTaskList, TaskManager tasksList, PatientManager patientList, Ui ui, PatientTaskStorage patientTaskStorage, TaskStorage taskStorage, PatientStorage patientStorage) throws DukeException {
+        this.newPatientTask = finalPatientTask(command);
         if (patientList.isExist(newPatientTask.getPatientId()) && tasksList.isExist(newPatientTask.getTaskID()))
```

Comment:
There is a bug for `taskList.isExist(newPatientTask.getTaskID())`.
The current `TaskManager.isExist(int i`) only check for if the existing number of tasks >= i, it doesn't serve the purpose of checking if a given task exists in the taskList.
Will fix this bug in the next PR by adding a `TaskManager.isTaskExisted(int taskID)` and remove the current `TaskManager.isExist`

Reply:
I suggest we can split the condition checking for `isPatientExisted `and `isTaskExisted`, such that when one of them return False, we can remind user specifically whether the Task or Patient doesn't exist.
This can be done as an improvement with an new story(As a nurse, I know which part of my command goes wrong...).